test/integration/contract.js

```diff
@@ -413,7 +413,7 @@ describe('Contract', function () {
     })
 
     it('registers', async () => {
-      delegationSignature = await aeSdk.delegateSignatureCommon([address, contractId])
+      delegationSignature = await aeSdk.createOracleDelegationSignature({ contractId })
```

Comment:
mhh, do we introduce another breaking change with this?

Reply:
Not a breaking change. `delegateSignatureCommon` or `createOracleDelegationSignature` any method can be used  and syntax remained same.

Reply:
I would switch to `delegateSignatureCommon` at the end because wrapper functions just places it's arguments into array, they are not needed.

Reply:
someone reading the client code it would be more pleasing if we keep `createOracleDelegationSignature`. It's more readable. having `delegateSignatureCommon` is very confusing.

Reply:
"entities should not be multiplied beyond necessity", `createAensDelegationSignature`, `createOracleDelegationSignature` just signing an array of arguments in the right order. I think user can pass them by himself, just according to documentation, like:
```
aens: address, name, contractId
oracle: queryId, contractId
```
Also, I don't like that `createOracleDelegationSignature` actually can be used to create AENS delegation signature and probably even to sign an arbitrary transaction.

But let's keep it as it is right now.